source/MRViewer/MRViewport.h

```diff
@@ -344,22 +344,18 @@ class Viewport
     {
         Visible, // fit all visible objects
         SelectedPrimitives, // fit only selected primitives
-        SelectedObjects // fit only selected objects
+        SelectedObjects, // fit only selected objects
+        CustomObjectsList // fit only given objects (need additional objects list)
     };
     struct FitDataParams
     {
-        float factor{ 1.f }; // part of the screen for scene location
+        float factor{ 0.9f }; // part of the screen for scene location
```

Comment:
Do we really need to change default value for everyone? Probably better to change the value in the new call place?

Reply:
removed this chenge